parsons/targetsmart/targetsmart_automation.py

```diff
@@ -9,75 +16,109 @@
 import xmltodict
 
 
-TS_STFP_HOST = 'transfer.targetsmart.com'
+TS_STFP_HOST = "transfer.targetsmart.com"
 TS_SFTP_PORT = 22
-TS_SFTP_DIR = 'automation'
+TS_SFTP_DIR = "automation"
 
 logger = logging.getLogger(__name__)
 
 # Automation matching documentation can be found here:
-# https://docs.targetsmart.com/developers/automation/index.html
+# https://docs.targetsmart.com/developers/automation/index.html. A custom
+# matching workflow has been implemented by TargetSmart for TMC.
```

Comment:
Can `TSAutomation` only be used by TMC? Not sure what this note means.

Reply:
We should certainly get everyone on the same page about this one - if it's just a note in docs about something that TMC does differently than others, it can be struck. If the TargetSmartAutomation class is only useful to TMC, we should probably strike it from the PR and integrate it into TMC work internally. Parsons is widely used both inside and outside TMC, and we try to avoid including anything in the repo that ties it specifically to a TMC use case.

Reply:
Hi all. TargetSmart Automation is a general solution for clients to execute custom workflows that are developed and deployed for specific client needs. Given that, it's a very general interface. For example, the details of the input and output files vary from workflow to workflow. We currently maintain a custom matching workflow for TMC organizations. Other TargetSmart clients also use Automation, but with different custom workflows/layouts/parameters than TMC orgs.